Datadog.Trace.sln

```diff
@@ -1248,6 +1250,10 @@ Global
 		{959E9599-8D99-43BC-8038-B91F76179C1C}.Debug|Any CPU.Build.0 = Debug|Any CPU
 		{959E9599-8D99-43BC-8038-B91F76179C1C}.Release|Any CPU.ActiveCfg = Release|Any CPU
 		{959E9599-8D99-43BC-8038-B91F76179C1C}.Release|Any CPU.Build.0 = Release|Any CPU
+		{95DEBF4E-6017-40AD-9840-C7D112591228}.Debug|Any CPU.ActiveCfg = Debug|Any CPU
+		{95DEBF4E-6017-40AD-9840-C7D112591228}.Debug|Any CPU.Build.0 = Debug|Any CPU
+		{95DEBF4E-6017-40AD-9840-C7D112591228}.Release|Any CPU.ActiveCfg = Release|Any CPU
+		{95DEBF4E-6017-40AD-9840-C7D112591228}.Release|Any CPU.Build.0 = Release|Any CPU
```

Comment:
```suggestion
```

Reply:
done in https://github.com/DataDog/dd-trace-dotnet/pull/4422/commits/01fcf69054a323668f0be7f36bed0fcfef6943ac